### PR TITLE
Update data-loss-prevention-policies.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox Please review the changes make edits if required. Could you do the formatting (numbering) adjustments.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 1]
```diff
@@ -141,7 +141,12 @@ When content matches a condition in a rule, you can apply actions to automatical
   
 With the actions now available, you can:
   
-- **Restrict access to the content** For site content, this means that permissions for the document are restricted for everyone except the primary site collection administrator, document owner, and person who last modified the document. These people can remove the sensitive information from the document or take other remedial action. When the document is in compliance, the original permissions are automatically restored. When access to a document is blocked, the document appears with a special policy tip icon in the library on the site. 
+- **Restrict access to the content** Depending on your need, you can restrict access to content in three ways
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
- **Restrict access to the content** Depending on your need, you can restrict access to content in three ways:
```

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 2]
```diff
@@ -141,7 +141,12 @@ When content matches a condition in a rule, you can apply actions to automatical
...
+1. Restrict everyone
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
1. Restrict access to content for everyone.
```

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 3]
```diff
@@ -141,7 +141,12 @@ When content matches a condition in a rule, you can apply actions to automatical
...
+2. Restrict people outside the organization
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
2. Restrict access to content for people outside the organization.
```

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 4]
```diff
@@ -141,7 +141,12 @@ When content matches a condition in a rule, you can apply actions to automatical
...
+3. Restrict access to "Anyone with the link"
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md:147]:
```suggestion
3. Restrict access to content for "Anyone with the link."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Comment by mypil** [comment]
@pratjain523 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by pratjain523** [comment]
I have committed 3 changes except for the one below. It should be Restrict access to content using “Anyone with the link” option.

[cid:image001.png@01D63B28.1B9E0600]

Thanks,
Prateek

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**